tpm_test.go

```diff
@@ -171,7 +171,7 @@ func createTestCA() ([]byte, crypto.PrivateKey, error) {
 }
 
 func createTestEkCert(tpm *tpm2.TPMContext, caCert []byte, caKey crypto.PrivateKey) ([]byte, error) {
-	ekContext, pub, _, _, _, err := tpm.CreatePrimary(tpm.EndorsementHandleContext(), nil, &EkTemplate, nil, nil, nil)
+	ekContext, pub, _, _, _, err := tpm.CreatePrimary(tpm.EndorsementHandleContext(), nil, EkTemplate, nil, nil, nil)
```

Comment:
I'm quite confused, do some/which of these functions modify the template in place? don't we have the same issue in the real code, that the template cananot be reused?

Reply:
There aren't any other functions that modify the template.